interface/src/app/gene/network/network.js

```diff
@@ -390,9 +401,16 @@ angular.module('adage.gene.network', [
                 return g.id;
               });
 
-              ExpressionValue.get(
-                {'gene__in': geneID.join(), 'sample__in': sampleID.join(),
-                  'order_by': 'gene'},
+              ExpressionValue.post(
+                {},
+                // Angular does not serialize POST data the way we would expect
+                // so we have to do it manually here. For more details, see
+                // https://github.com/angular/angular.js/issues/6039#issuecomment-113502695
+                $httpParamSerializerJQLike({
```

Comment:
👍 

The discussion at Angular issue 6039 was more enlightening for me than the documentation. It seems there are some on the Angular team who agree that the framework could do more to help here and others who seem to prefer that "correcting" the default behavior would break too many things. It seems that we are left to do this sort of manual patch for the foreseeable future.